2019/d02p1IntCode.d

```diff
@@ -78,4 +80,21 @@ unittest {
   parse(8, i);
   assert(!execute(i));
   assert(memory == [30,1,1,4,2,5,6,0,99]);
+} version (unittest) {} else {
```

Comment:
What is this?

Reply:
This lets me run the unit tests without the main function using `rdmd -unittest -main` or run the main function if the unit tests pass by omitting `-main`. The [version](https://dlang.org/spec/version.html) feature has so much more but I love it for this alone. It helps me evolve working code.